### ISSUE TITLE
Steam Guard for Chinese

### ISSUE THREAD

**Issue by krnl0x0** [rex706/SAM#83]
As in issue #25 I find that in Chinese, the "Steam Guard" is translated to "Steam 令牌", which will cause SAM fail to locate the window.

And also, when typing the 2FA code, it will trigger the IME working, but that's a small issue.

**Comment by rex706** [comment]
@krnl0x0 what do you mean by "trigger the IME working"?

**Comment by krnl0x0** [comment]
@rex706 It means that the Microsoft IME for Chinese is enabled by default, therefore when SAM start typing 2FA code, IME will catch and treat them as Chinese "syllable" instead of normal English inputs.

That's not a big deal, because the IME can be disabled. And the only way I find to bypass it, is to enable the CapsLock, I've implemented using this method via https://github.com/krnl0x0/SAM/commit/81de00339853bfed0b573723e50626d7952143e3, for your convinent.

**Comment by rex706** [comment]
@krnl0x0 do you know if this is an issue if `SendMessage()` is used isntead of `SendWait()`? Does Caps-Lock need to be toggled in the case of using `SendMessage()` in `MainWindow.xaml.cs`?

**Comment by krnl0x0** [comment]
@rex706 I've tested and it seems totally fine with `SendMessage`, even without CapsLock. So, nice work!

I tried to look up the MSDN, and found

> For a non-Unicode window, if the WM_IME_CHAR message includes a double-byte character and the application passes this message to DefWindowProc, the IME converts this message into two WM_CHAR messages, each containing one byte of the double-byte character.

in [WM_IME_CHAR](https://docs.microsoft.com/en-us/windows/win32/intl/wm-ime-char), so it means that IME will produce the `WM_CHAR` eventually for system to handle. In another word, if we send `WM_CHAR` message, then we can actually bypass the IME.

And you might need to revert the changes that I made ;)
